CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ OceanBase Database is a community-driven open source project and we welcome all
 
 ## Before you contribute
 
-Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/pingcap/tidb?pullRequest=16303).
+Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/oceanbase/oceanbase).
```

Comment:
```suggestion
Before you contribute, please click the **Sign in with GitHub to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/oceanbase/oceanbase).
```
The `h` in `GitHub` should be capitalized.
![image](https://user-images.githubusercontent.com/8433465/120261179-ef10c200-c2c9-11eb-9e5b-dd689d46d8db.png)

Reply:
Good catch!

Reply:
![image](https://user-images.githubusercontent.com/29879298/120261425-74947200-c2ca-11eb-8f94-ea00aeaf6a2d.png)

It should be capitalized.